user/nmap/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "nmap"
+pkgver = "7.95"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--with-openssl",
+    "--without-zenmap",
+    "--without-ndiff",
+    "--without-ncat",
```

Comment:
```suggestion
```

no reason to disable these

Reply:
Part of the ncat test requires doing a dns resolution. I'm guessing this means that the ncat tests cannot be run since the cbuild sandbox has no network access during the check phase. Assuming that's true, is there a natural/obvious way to disable tests more granularly than `options = ["!check"]`?

Reply:
> Assuming that's true, is there a natural/obvious way to disable tests more granularly than options = ["!check"]?  

if the build system exposes one, sure. if not, then no. dunno if there is a way in this case